tools/cloud-build/daily-tests/ansible_playbooks/tasks/gather_startup_script_logs.yml

```diff
@@ -16,18 +16,28 @@
 - name: Assert variables are defined
   ansible.builtin.assert:
     that:
-    - terraform_apply_stderr_one_line is defined
+    - ghpc_stderr is defined
 
+- name: Get serial port command
+  failed_when: false
+  ansible.builtin.set_fact:
+    serial_port_cmd: '{{ ghpc_stderr | regex_findall("please run:\s+(.+?\s+--project\s+\S+)", "\\1") | first }}'
```

Comment:
What is the value of serial_port_cmd when regex is not found? What does `first` produce when list is empty ? Seems like it could cause error.

I want to be sure that 1) case that nothing is found does not throw error and 2) case that nothing is found results in `serial_port_cmd is defined` resolving to false.

Reply:
I agree we should make sure all of this doesn't throw an error.  Alternatively, I'd propose that every task here simply uses the `failed_when: false` line to make sure we just keep working through since this is just logging, and we definitely want to get to the cleanup sections.  

The regex here isn't much different than the original line, so this shouldn't be a regression if I update the print out with `failed_when: false`

Reply:
I've updated this to include `| default("")` so if nothing is found, it will default to an empty string.  The next two tasks will check for an empty string and treat that as a failure.  This should improve the error handling above what was there before this PR.